src/Parse/Ini.php

```diff
@@ -202,7 +202,7 @@ protected function renderProperties($vars = [])
                     }
                 }
             }
-            elseif (strlen($value)) {
+            elseif ($value . '' !== '') {
```

Comment:
Why was this change made?

Reply:
This fails the `testBasic` see [here](https://github.com/octobercms/library/blob/develop/tests/Parse/IniTest.php#L19) - The [render ](https://github.com/octobercms/library/blob/develop/tests/Parse/IniTest.php#L40) method is passed values that aren't string. 
Again this error only happens if you have `declare(strict_types=1);` declared in your `October\Rain\Parse\Ini.php` file which I understand is not a coding standard for this library but thought this may be useful as a side note.

As for the work around maybe add `(strlen((string) $value))` or whatever you think is appropriate.

Which also takes to question the `evalValue` method - It is supposed to return a string and why?

        // Numeric - This doesn't return a string as the value is returned as it is.
        if (is_numeric($value)) {
            return $value;   
        }

test screenshot

![1](https://user-images.githubusercontent.com/4506434/92621522-868c3b80-f289-11ea-8536-2584518246dc.png)